client/elements/sc-navdrawer.html

```diff
@@ -283,7 +284,7 @@ <h2 class="menu-header">Publication details</h2>
           }
 
           _getUrl() {
-              return `${this.$.meta.byKey('API_ROOT')}/menu`;
+              return `${this._getApiUrl()}/menu`;
```

Comment:
I take it `${this.$.meta.byKey('API_ROOT')}/menu` didn't work here?

Reply:
that's correct. it was undefined here.